### DISCUSSION TITLE
Feature request: Suggest timezone based on last-chosen timezone

### DISCUSSION THREAD

**Discussion by mtlynch** [plausible/analytics#1074] [Anything Else]
I'm in Massachusetts, which is US Eastern Time. When I add a site, Plausible auto-suggests Atlantic Time (Canada), which I'm guessing is based on imprecise IP geolocation (understandable). But it seems to not learn from my previous correction. I'm 99% sure I selected Eastern Time on my first site (I don't see any way to verify that, though), but Plausible is suggesting Atlantic Time for my second site.

![image](https://user-images.githubusercontent.com/7783288/119231484-2b386b80-baef-11eb-9d02-29b499467559.png)

It'd be cool if Plausible guessed that the time zone for a user's site N was going to be the same time zone as site N-1.

**Comment by ukutaht** [comment]
Great idea, thanks for the suggestion
